content/project/adopters.html.haml

```diff
@@ -0,0 +1,38 @@
+---
+layout: simplepage
+title: "Jenkins Adopters"
+section: project
+description: "List of Jenkins adopters"
+---
+
+This page provides a list of Jenkins adopters.
+
+.grid-container    
+  - site.adopters.keys.each do |p|
+    = partial("company-card.html.haml", :companyId => p)
+
+%h1 More adopters
+
+You can find more Jenkins adopters listed on its StackShare page.
+According to this site, Jenkins has more than 2500 adopter companies which you can discover using its web interface and API.
+%a{:href => "https://stackshare.io/jenkins"}
+  Jenkins on Stackshare
+
+%p
+  %a{:href => "https://stackshare.io/jenkins"}
+    %img{:src => expand_link("images/adopters/stackshare.png"), :alt => "Adopters on Stackshare", :width => "100%"}
+
+%h1 References
+
+%ul
+  %li
+    %a{:href => "https://stackshare.io/jenkins"}
+      Jenkins on Stackshare
+  %li
+    %a{:href => "https://jenkinsistheway.io/"}
+      JenkinsIsTheWay portal - more case studies
+  %li
+    %a{:href => "https://github.com/jenkins-infra/jenkins.io/tree/master/content/_data/authors"}
```

Comment:
I think the "Add your company" link should navigate to the Adopters page rather than the Authors page, though I'm OK with either.  Will entries in both be required when someone adds their company as a Jenkins adopter?
```suggestion
    %a{:href => "https://github.com/jenkins-infra/jenkins.io/tree/master/content/_data/adopters"}
```